docs/components/paragraph.md

```diff
@@ -0,0 +1,73 @@
+# matestack core component: Paragraph
+
+Show [specs](../../spec/usage/components/paragraph_spec.rb)
+
+The HTML `<p>` tag implemented in ruby. This is a workaround because the single `p` is the short version of `puts` in ruby.
```

Comment:
@PasWen  actually `p` is a bit different from `puts`, it's not the short version. `p` calls `.inspect` on the printed object, `puts` not.

Reply:
Hey,
thanks for pointing it out, just fixed it in the documentation (and learned sth new on the way;)!